Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -435,6 +436,8 @@ extension TelemetryWrapper {
             GleanMetrics.Tabs.open[value].add()
         case (.action, .close, .tab, let value, _):
             GleanMetrics.Tabs.close[value].add()
+        case (.action, .tap, .addNewTabButton, _, _):
+            GleanMetrics.Tabs.addNewTabPressed.add()
```

Comment:
**Comment A**
This won't compile until `Metrics.swift` is not updated. Will have to ping Travis.